nvme-print-json.c

```diff
@@ -4701,14 +4708,16 @@ static void json_reachability_groups_log(struct nvme_reachability_groups_log *lo
 	json_print(r);
 }
 
-static void json_reachability_associations_log(struct nvme_reachability_associations_log *log)
+static void json_reachability_associations_log(struct nvme_reachability_associations_log *log,
+					       __u64 len)
 {
 	struct json_object *r = json_create_object();
 	__u16 i;
 	__u32 j;
 	char json_str[STR_LEN];
 	struct json_object *rad;
 
+	obj_add_debug(uint64, r, "len", len);
```

Comment:
I am not sure if we should add debug output to the json output. For debugging purposes we have the normal output.

Reply:
Fixed to delete the debug output but added to use UNUSED macro instead since the len parameter not used.